tests/test_isamples_metadata.py

```diff
@@ -85,6 +88,10 @@ def test_geome_child_dicts_equal(geome_source_path, isamples_path):
     (
         "../examples/OpenContext/raw/ark-28722-k2m61xj9b.json",
         "../examples/OpenContext/test/ark-28722-k2m61xj9b-test.json"
+    ),
+    (
+        "../examples/OpenContext/raw/ark-28722-k2qj7np9g.json",
+        "../examples/OpenContext/test/ark-28722-k2m61xj9b-test.json"
```

Comment:
Unrelated cleanup -- I had this in my local copy and apparently forgot to check it in when working on OpenContext